From ed68bb0f2ed5b1f839e7dc13f040feeae05ba4ba Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Sun, 30 Jun 2013 23:17:49 +0300
Subject: [PATCH 255/290] be: ask a question in tx_engine_got_closed()

.c files may be not a proper media for asking questions, but Nikita
is offline at the moment. ;)
---
 be/tx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/be/tx.c b/be/tx.c
index 9da9d19..ced1eb1 100644
--- a/be/tx.c
+++ b/be/tx.c
@@ -499,6 +499,7 @@ tx_engine_got_closed(struct m0_be_tx_engine *eng, struct m0_be_tx *tx)
 		else
 			tx_group_add(eng, gr, tx);
 	}
+	/* XXX ANSWERME: What do we do if the group is closed? --vvv */
 #else /* XXX: close group after every tx close, for now */
 	tx_group_init(gr);
 	tx_group_add(eng, gr, tx);
-- 
1.8.3.2

